src/rsz/test/remove_buffers3.ok

```diff
@@ -0,0 +1,46 @@
+[INFO ODB-0227] LEF file: Nangate45/Nangate45.lef, created 22 layers, 27 vias, 135 library cells
+[INFO ODB-0128] Design: top
+[INFO ODB-0130]     Created 2 pins.
+[INFO ODB-0131]     Created 4 components and 16 component-terminals.
+[INFO ODB-0132]     Created 2 special nets and 8 connections.
+[INFO ODB-0133]     Created 5 nets and 8 connections.
+Startpoint: in1 (input port)
+Endpoint: out1 (output port)
+Path Group: unconstrained
+Path Type: max
+
+  Delay    Time   Description
+---------------------------------------------------------
+   0.00    0.00 v input external delay
+   0.00    0.00 v in1 (in)
+   0.02    0.02 v b1/Z (BUF_X1)
+   0.02    0.05 v b2/Z (BUF_X2)
+   0.02    0.07 v b3/Z (BUF_X4)
+   0.02    0.09 v b4/Z (BUF_X8)
+   0.00    0.09 v out1 (out)
+           0.09   data arrival time
+---------------------------------------------------------
+(Path is unconstrained)
+
+
+[INFO RSZ-0026] Removed 2 buffers.
+Startpoint: in1 (input port)
+Endpoint: out1 (output port)
+Path Group: unconstrained
+Path Type: max
+
+  Delay    Time   Description
+---------------------------------------------------------
+   0.00    0.00 v input external delay
+   0.00    0.00 v in1 (in)
+   0.02    0.02 v b2/Z (BUF_X2)
+   0.02    0.04 v b4/Z (BUF_X8)
+   0.00    0.04 v out1 (out)
+           0.04   data arrival time
+---------------------------------------------------------
+(Path is unconstrained)
+
+
+Differences found at line 8.
+    - b2 BUF_X2 ;
+    - b2 BUF_X2 + PLACED ( 200 200 ) N ;
```

Comment:
There should be no differences in the ok file

Reply:
Hmm... Regression showed it as "passing" despite this difference.   save_ok didn't seem to update defok files.   This is a new testcase.  Updated both defok and ok now.